src/Datadog.Trace/Tracer.cs

```diff
@@ -477,6 +474,18 @@ private void RunShutdownTasks()
             TracingProcessManager.StopProcesses();
         }
 
+        private void DecorateRootSpan(Span span)
+        {
+            if (AzureAppServicesMetadata.IsRelevant())
+            {
+                var azureAppServiceResourceId = AzureAppServicesMetadata.GetResourceId();
+                if (azureAppServiceResourceId != null)
+                {
+                    span.SetTag(Tags.AzureAppServicesResourceId, azureAppServiceResourceId);
```

Comment:
@vlad-mh The tag would be on the root span of a trace only. (This is local traces, not distributed traces)